sass/_aesop.scss

```diff
@@ -1,375 +1,392 @@
-        }
-	}
-    
-    &-stop {
-        padding-top: 140px;
+  background-color: rgba(255, 255, 255, .9);
+  bottom: 0;
+  box-shadow: 0 0 1px rgba(0, 0, 0, .15);
+  left: 0;
+  position: fixed;
+  right: 0;
+  z-index: $z-index-aesop-timeline;
+
+  ol {
+    list-style: none;
+    margin-bottom: 0;
+    text-align: center;
+    @include site-width($width-wide);
+    &:after {
+      display: none;
     }
+  }
+
+  li {
+    display: inline-block;
+    margin: 0 10px;
+    transition: all 200ms;
+
+    a:hover {
+      border-color: $link-border-color;
+    }
+
+    &.active a {
+      border-bottom-width: 2px;
+      font-weight: bold;
+    }
+  }
+
+  &-stop {
+    padding-top: 140px;
+  }
 }
 
 .aesop-sticky-map.aesop-sticky-map-left .aesop-map-component:before {
-	box-shadow: none;
+  box-shadow: none;
 }
 
 .entry-content > .aesop-component,
 .entry-content > .aesop-story-collection {
-	max-width: 100%;
-	width: 100%;
-	
-	&:first-child {
-		margin-top: 0;
-	}
-	
-	&.aesop-audio-component {
-		@include site-width;
-	}
+  max-width: 100%;
+  width: 100%;
+
+  &:first-child {
+    margin-top: 0;
+  }
+
+  &.aesop-audio-component {
+    @include site-width;
+  }
 }
 
 #aesop-toggle-chapter-menu,
 #aesop-chapter-menu {
-	display: none; // fix meeeee
+  display: none; // fix meeeee
 }
 
 .aesop-grid {
-    &-image {
-        width: inherit; // overrides inline style/property
+  &-image {
+    width: inherit; // overrides inline style/property
+  }
+
+  &-gallery {
+    .aesop-grid-gallery-caption {
+      @include rem(font-size, 16px);
+      background-color: transparent;
+      background-image: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, .5));
+      bottom: 0;
+      color: #fff;
+      left: 0;
+      padding: 40px 20px 20px;
+      right: 0;
+      top: auto;
+      z-index: 1; // keeps caption above image
     }
-    
-    &-gallery {
-        .aesop-grid-gallery-caption {
-            @include rem(font-size, 16px);
-            background-color: transparent;
-            background-image: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, .5));
-            bottom: 0;
-            color: #fff;
-            left: 0;
-            padding: 40px 20px 20px;
-            right: 0;
-            top: auto;
-            z-index: 1; // keeps caption above image
-        }
-        
-        &-item {
-            text-align: center;
-        }
+
+    &-item {
+      text-align: center;
     }
+  }
 }
 
 .fotorama {
-    &__caption__wrap {
-        @include rem(font-size, 16px);
-        background-color: transparent;
-        background-image: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, .5));
-        color: #fff;
-        display: block;
-        padding: 80px 20px 20px;
-    }
-    
-    &__thumb-border {
-        border-color: $color-blue;
-    }
+  &__caption__wrap {
+    @include rem(font-size, 16px);
+    background-color: transparent;
+    background-image: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, .5));
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.